ui-v2/app/mixins/with-resizing.js

```diff
@@ -17,14 +17,22 @@ export default Mixin.create({
   },
   didInsertElement: function() {
     this._super(...arguments);
-    get(this, 'win').addEventListener('resize', this.handler);
+    ['resize', 'orientationchange'].forEach(
```

Comment:
Orientation change doesn't also invoke `resize`? 😧

Reply:
Well, weirdly I seem to remember it being problematic, and I vaguely remember in a past life this catching me out. I've had a quick snoop around and I can find references to it being problematic but mainly on older and/or non-targeted OS's - just not really enough good evidence around that.

I did some more testing myself and  from what I can see resize is being fired on orientation change, so I've removed this and reverted back to just `resize`.